octobot_commons/os_clock_sync.py

```diff
@@ -39,11 +39,13 @@ def _get_sync_cmd(self):
         platform = os_util.get_os()
         bot_type = os_util.get_octobot_type()
         if bot_type == commons_enums.OctoBotTypes.DOCKER.value:
-            raise NotImplementedError(bot_type)
+            return "service ntp stop && ntpd -gq && service ntp start"
```

Comment:
There is no ntp service in the current octobot docker image

Reply:
I added it to the dockerfile as well see PR in octobot -> [Docker] add ntp to docker version

Reply:
I don't think we should add ntp service in docker based on this thread: https://stackoverflow.com/questions/24551592/how-to-make-sure-dockers-time-syncs-with-that-of-the-host

Reply:
Oh yeah crap, I removed the docker part, now only the linux root fix is included